client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -43,13 +43,13 @@ export default React.createClass( {
 		);
 	},
 
-	getFormHeader() {
+	renderFormHeader() {
 		return (
 			<InviteFormHeader { ...this.props } />
 		);
 	},
 
-	loginUser() {
```

Comment:
I choose this name because it is more like an action, as the form will never get shown, this will render the form and autosubmit it in order to login the user. I don't mind changing it if you feel it makes more sense :)